src/Adapter/PlatformAdapter/Native/Sdk/VggSdkNodeAdapter.cpp

```diff
@@ -292,6 +294,40 @@ napi_value VggSdkNodeAdapter::UpdateElement(napi_env env, napi_callback_info inf
   return nullptr;
 }
 
+napi_value VggSdkNodeAdapter::setCurrentFrame(napi_env env, napi_callback_info info)
+{
+  size_t     argc = 1;
+  napi_value args[1];
+  napi_value _this;
+  NODE_API_CALL(env, napi_get_cb_info(env, info, &argc, args, &_this, NULL));
+
+  if (argc != 1)
+  {
+    napi_throw_error(env, nullptr, "Wrong number of arguments");
+    return nullptr;
+  }
+
+  try
+  {
+    auto name = GetArgString(env, args[0]);
```

Comment:
Maybe the parameter type should be an integer? Because frame name strings could be the same, and the frame setting could be unexpected.

Reply:
Added `setCurrentFrameById`.